src/sentry/models/counter.py

```diff
@@ -22,22 +23,28 @@ def increment(cls, project, delta=1):
         return increment_project_counter(project, delta)
 
 
-def increment_project_counter(project, delta=1):
+def increment_project_counter(project, delta=1, using="default"):
     """This method primarily exists so that south code can use it."""
     if delta <= 0:
         raise ValueError("There is only one way, and that's up.")
 
-    cur = connection.cursor()
-    try:
-        cur.execute(
-            """
-            select sentry_increment_project_counter(%s, %s)
-        """,
-            [project.id, delta],
-        )
-        return cur.fetchone()[0]
-    finally:
-        cur.close()
+    # To prevent the statement_timeout leaking into the session we need to use
```

Comment:
Please let's add a visible warning that this is not a proper fix and should be removed when we figure out a better way to get the next project id.

Reply:
Added warning next to the `statement_timeout` line.